modules/acknowledgements/php/NDB_Menu_Filter_Form_acknowledgements.class.inc

```diff
@@ -225,7 +225,7 @@ class NDB_Menu_Filter_Form_Acknowledgements extends NDB_Menu_Filter_Form
         return array_merge(
             $baseDeps,
             array(
-             $baseurl . '/acknowledgements/js/acknowledgements.js',
+             $baseurl . '/acknowledgements/js/acknowledgements_helper.js',
```

Comment:
Yes, so there is no confusion with what gets automagically loaded.  Now that the files are not identical to the module name, they will not trigger the warning in the logs.